pages/learn/deploy/offline-updates.md

```diff
@@ -40,15 +40,22 @@ To perform an offline update, we will be using [balena-cli][balena-cli]. All com
 
 Offline update includes the following steps:
 
-- [Setup](#setup)
-- [Create/use pre-existing application](#createuse-pre-existing-application)
-- [Create/use pre-existing offline device](#createuse-pre-existing-offline-device)
-- [Download balenaOS image](#download-balenaos-image)
-- [Configure balenaOS image](#configure-balenaos-image)
-- [Create and preload release](#create-and-preload-release)
-- [Create update media](#create-update-media)
-- [Process of reprovisioning](#process-of-reprovisioning)
-- [Update device registration(s)](#update-device-registrations)
+- [Offline Updates](#offline-updates)
```

Comment:
This table looks strange - is it automatically generated?

Reply:
No that's my bad, vscode auto linted this into a TOC
Fixing

Reply:
Resolving